docker-compose.yml

```diff
@@ -1,15 +1,15 @@
-version: '3.7'
+name: amsterdam-bereikbaarheid
 
 services:
-  frontend:
+  app:
     build: &build-config
       context: .
       args:
-        - BUILD_ENV=development
-        - NGINX_CONF=nginx.dockercompose.conf
+        BUILD_ENV: prod
```

Comment:
Write the env in full? production

Reply:
Be aware that this is the BUILD_ENV and not Node's NODE_ENV. The latter one is fully written out. This one wasn't (legacy). This short name corresponds to `.env.prod`.

Reply:
BUILD_ENV is verwijderd. Gaat geregeld worden vanuit de infra pipelines.